README.md

```diff
@@ -93,21 +92,25 @@ nginx::resource::upstream { 'mattermost':
   members => [ 'localhost:8065' ],
 }
 nginx::resource::vhost { 'mattermost':
-  server_name => [ 'myserver.mydomain' ],
-  proxy       => 'http://mattermost',
+  server_name         => [ 'myserver.mydomain' ],
+  proxy               => 'http://mattermost',
+  location_cfg_append => {
+    'proxy_http_version'          => '1.1',
+    'proxy_set_header Upgrade'    => '$http_upgrade',
+    'proxy_set_header Connection' => '"upgrade"',
+  },
```

Comment:
Added to the location config per the [NGINX documentation on websockets](https://www.nginx.com/blog/websocket-nginx/), [Mattermost docs](http://docs.mattermost.com/install/prod-debian.html#set-up-nginx-server), and [Mattermost forums](http://forum.mattermost.org/t/setup-behind-nginx/86). This prevents some common Websocket errors.